interface-impl-amf/src/main/java/org/mule/amf/impl/model/JsonParameterValidationStrategy.java

```diff
@@ -6,31 +6,35 @@
  */
 package org.mule.amf.impl.model;
 
-import amf.client.model.domain.ValidatorAware;
-import amf.client.validate.PayloadValidator;
-import amf.client.validate.ValidationReport;
+import amf.apicontract.client.platform.AMFElementClient;
+import amf.core.client.common.validation.ValidationMode;
+import amf.core.client.platform.validation.AMFValidationReport;
+import amf.core.client.platform.validation.payload.AMFShapePayloadValidator;
+import amf.shapes.client.platform.model.domain.AnyShape;
 import org.json.simple.JSONValue;
-import org.mule.amf.impl.exceptions.ParserException;
 import org.mule.amf.impl.util.LazyValue;
 
 import static org.mule.amf.impl.model.MediaType.APPLICATION_JSON;
 import static org.mule.amf.impl.model.MediaType.APPLICATION_YAML;
 
 class JsonParameterValidationStrategy extends ValidationStrategy {
 
-  private final LazyValue<PayloadValidator> jsonValidator =
-      new LazyValue<>(() -> schema.payloadValidator(APPLICATION_JSON)
-          .orElseThrow(() -> new ParserException(APPLICATION_JSON + " validator not found for shape " + schema)));
+  private AMFElementClient client;
```

Comment:
Do we need to keep a reference to the client here or only on the LazyValue (like on the YamlParameterValidationStrategy).